input/drivers/udev_input.c

```diff
@@ -204,16 +207,23 @@ enum udev_input_dev_type
    UDEV_INPUT_KEYBOARD = 0,
    UDEV_INPUT_MOUSE,
    UDEV_INPUT_TOUCHPAD,
-   UDEV_INPUT_TOUCHSCREEN
+   UDEV_INPUT_TOUCHSCREEN,
+   UDEV_INPUT_SENSOR
 };
-
+static bool udev_is_mouse(enum udev_input_dev_type type){
+   return (
+      (type == UDEV_INPUT_MOUSE) ||
+      (type == UDEV_INPUT_TOUCHPAD) || 
+      (type == UDEV_INPUT_TOUCHSCREEN));
+}
 /* NOTE: must be in sync with enum udev_input_dev_type */
 static const char *g_dev_type_str[] =
 {
    "ID_INPUT_KEY",
    "ID_INPUT_MOUSE",
    "ID_INPUT_TOUCHPAD",
-   "ID_INPUT_TOUCHSCREEN"
+   "ID_INPUT_TOUCHSCREEN",
+   "ID_INPUT_ACCELEROMETER"
```

Comment:
Should there be `ID_INPUT_GYROSCOPE` as well? Or am I misunderstanding something?

Reply:
"ID_INPUT_ACCELEROMETER" is the whole sensor device according to udev